src/ripple/protocol/STTx.h

```diff
@@ -108,8 +108,16 @@ class STTx final : public STObject, public CountedObject<STTx>
 
     Json::Value
     getJson(JsonOptions options) const override;
+
+    /// If `hash` is set, will store hash inside the provided string. Otherwise
```

Comment:
Is it more accurate to say -- "If `hash` is set, will store transaction ID inside the provided string" ?

Reply:
thanks , will do that !